.bazelrc

```diff
@@ -130,11 +130,13 @@ build:release-debug-mac --config=release-debug-common
 build:release-sanitized-linux --copt=-O2
 build:release-sanitized-linux --config=sanitize-linux
 build:release-sanitized-linux --define jemalloc=false
+build:release-sanitized-linux --define mimalloc=false
 build:release-sanitized-linux --config=release-debug-common
 
 build:release-sanitized-mac --copt=-O2
 build:release-sanitized-mac --config=sanitize-mac
 build:release-sanitized-mac --define jemalloc=false
+build:release-sanitized-mac --define mimalloc=false
```

Comment:
Can we look at the blame for the lines that added jemalloc=false and see if they mentioned why they were marked that way? Is it possible that we could actually keep mimalloc turned on when we run sanitized builds now?

Reply:
ack